scania-digital-design-system/sdds
# Feature - Big updated colours

**Describe pull-request**  
_Describe what the pull-request is about_
Updated colours

**Solving issue**  
_Add which issue this pull-request solves by adding # plus the number of the issue (for example #123)_
Fixes: [AB#944](https://dev.azure.com/scaniadigitaldesignsystem/3d9eb9c6-0045-473d-bc1d-e842dd779200/_workitems/edit/944)

**How to test**  
_Add description how to test if possible_
1. Go to storybook preview 
2. Check in colours 
3. Check components with the new colours and compare with figma


**Additional context**  
_Add any other context about the pull-request here._
There are some colours/state missing on some components, notes done already
